src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -376,14 +376,27 @@ private void generateSummarySpreadsheet(
 
     Pair<String, StatusForRound>[] statusesToPrint =
         new Pair[] {
-          new Pair<>("Overvotes", StatusForRound.INACTIVE_BY_OVERVOTE),
-          new Pair<>("Skipped Rankings", StatusForRound.INACTIVE_BY_SKIPPED_RANKING),
-          new Pair<>("Exhausted Choices", StatusForRound.INACTIVE_BY_EXHAUSTED_CHOICES),
-          new Pair<>("Repeated Rankings", StatusForRound.INACTIVE_BY_REPEATED_RANKING)
+          new Pair<>("Inactive Ballots by Overvotes",
+                     StatusForRound.INACTIVE_BY_OVERVOTE),
+          new Pair<>("Inactive Ballots by Skipped Rankings",
+                     StatusForRound.INACTIVE_BY_SKIPPED_RANKING),
+          new Pair<>("Inactive Ballots by Repeated Rankings",
+                     StatusForRound.INACTIVE_BY_REPEATED_RANKING),
+          new Pair<>("Inactive Fully Ranked Ballots by Exhausted Choices",
+                     StatusForRound.INACTIVE_BY_EXHAUSTED_CHOICES),
+          new Pair<>("Inactive Partially Ranked Ballots by Exhausted Choices",
+                     StatusForRound.INACTIVE_BY_UNUSED_RANKINGS),
         };
 
     for (Pair<String, StatusForRound> statusToPrint : statusesToPrint) {
-      csvPrinter.print("Inactive Ballots by " + statusToPrint.getKey());
+      // Skip any "Inactive by X" if no ballots are associated with that bucket
+      RoundTally lastRound = roundTallies.get(numRounds);
+      BigDecimal numUndervotesLastRound = lastRound.getBallotStatusTally(statusToPrint.getValue());
+      if (numUndervotesLastRound.equals(BigDecimal.ZERO)) {
+        continue;
```

Comment:
I think the summary_extended.csv split will solve this problem. I worry that consumers could be confused if they don't see explicit 0's.

Reply:
The alternative is the following pseudocode:

```
doShowPartiallyRankedInactive = config.userHasProvidedAMaxRanking()
foreach (CVRSource source) {
  if (source.doesThisSourceOverrideConfigsMaxRanking) {
    doShowPartiallyRankedInactive = true;
  }
}
```

But keep in mind, with this logic in a multi-source contest, only some sources will trigger this. If the config is set to MAX, a Dominion config is set to 6, and an ES&S config leaves the max unspecified, only ballots from Dominion will end up in this bucket.

Should we prevent one source from having a MAX set and another not having it set? Should we change language to clarify this possibility?